node/task.ts

```diff
@@ -1775,6 +1803,218 @@ export class CodeCoverageEnabler {
+ * to the current timeline record.
+ * 
+ * The file shall be available for download along with task logs.
+ * 
+ * @param containerFolder   Folder that the file will upload to, folder will be created if needed.
+ * @param name              Artifact name.
+ * @param path              Path to the file that should be uploaded.
+ * @returns                 void
+ */
+export function uploadArtifact(containerFolder: string, name: string, path: string) {
+    command("artifact.upload", { "containerfolder": containerFolder, "artifactname": name }, path);
+}
+
+/**
+ * Upload user interested log to build’s container “logs\tool” folder.
+ * 
+ * @param path      Path to the file that should be uploaded.
+ * @returns         void
+ */
+export function uploadBuildLog(path: string) {
+    command("build.uploadlog", null, path);
+}
+
+/**
+ * Update build number for current build.
+ * 
+ * @param value     Value to be assigned as the build number.
+ * @returns         void
+ */
+export function updateBuildNumber(value: string) {
+    command("build.updatebuildnumber", null, value);
+}
+
+/**
+ * Add a tag for current build.
+ * 
+ * @param value     Tag value.
+ * @returns         void
+ */
+export function addBuildTag(value: string) {
+    command("build.addbuildtag", null, value);
+}
+
+/**
+ * Update release name for current release.
+ * 
+ * @param value     Value to be assigned as the release name.
+ * @returns         void
+ */
+export function updateReleaseName(name: string) {
+    assertAgent("2.132");
+    command("build.updatereleasename", null, name);
+}
+
+/**
+ * Create an artifact link, artifact location is required to be 
+ * a file container path, VC path or UNC share path. 
+ * 
+ * The file shall be available for download along with task logs.
+ * 
+ * @param name              Artifact name.
+ * @param path              Path to the file that should be associated.
+ * @param artifactType      ArtifactType enum of Container, FilePath, VersionControl, GitRef or TfvcLabel.
+ * @returns                 void
+ */
+export function associateArtifact(name: string, path: string, artifactType: ArtifactType) {
+    command("artifact.associate", { "type": artifactType, "artifactname": name }, path);
+}
+
+/**
+ * Instruction for the agent to update the PATH environment variable.
+ * The specified directory is prepended to the PATH.
+ * The updated environment variable will be reflected in subsequent tasks.
+ * 
+ * @param path      Local directory path.
+ * @returns         void
+ */
+export function prependPath(path: string) {
+    assertAgent("2.115.0");
+    command("task.prependpath", null, path);
+}
+
+/**
+ * Upload and attach summary markdown to current timeline record.
+ * This summary shall be added to the build/release summary and 
+ * not available for download with logs.
+ * 
+ * @param path      Local directory path.
+ * @returns         void
+ */
+export function uploadSummary(path: string) {
+    command("task.uploadsummary", null, path);
```

Comment:
```assertAgent("0.5.6");``` https://github.com/Microsoft/azure-pipelines-tasks/blob/master/docs/authoring/commands.md

Reply:
The problem is that you can't assert the agent version lower than 2.104.1
https://github.com/Microsoft/azure-pipelines-task-lib/blob/9fdd82cd724c61aa1615e732a20d3a5ffce67397/node/task.ts#L90

Reply:
Oh whoops, my mistake. Good call